### ISSUE TITLE
Better educate by increasing the quality and audience of our talks

### ISSUE THREAD

**Issue by MarkBennett** [ExchangeJS/exchangejs.github.io#51]
The possible ways we've come up with to achieve this objective include:
- [ ] provide a short speaker guide
- [ ] provide talk templates (reveal.js w/ Markdown, Google Slides)
- [ ] share a talk schedule with at least two talks for the length of the year
- [ ] provide 10 minutes of newbie content at each meetup
- [ ] record and share talks (#28)
- [ ] enforce time limits for all talks
- [ ] provide speaker's practice and feedback
- [ ] poll members for speaker ideas
- [ ] track metrics for talk quality and audience size
- [ ] OTHERS?

**Comment by CAYdenberg** [comment]
I think we should prepare a short "Guide for Speakers" that addresses things like:
1. suggested length (# min and # of slides)
2. characteristics of the audience (gathered from our survey)
3. general notes on slide design (descriptive titles, keep code samples minimal)
4. general notes on speaking (encourage practising beforehand)

In addition, I think it would be worth segmenting the different kinds of talks we get so we can decide what we'd like to see more of/less of. Of the top of my head:
1. Beginner talks (functions, objects)
2. More advanced general topics (design patterns, unit testing)
3. Tooling (TypeScript, gulp, babel)
4. Frameworks/libraries (Ember, React, Ionic, Meteor)
5. General web stuff (web components, HTTP2)

Feel free to add to this. My gut feeling is that we need more of (2), and that even talks falling into (3) and (4) could spend more time in (2) territory (ie address what problem the tool or framework solves first, then introduce it or show something built with it).

**Comment by MarkBennett** [comment]
Those ideas are perfect @CAYdenberg, and I really like your suggestion of categorizing the talks so we're not dominated by or missing any particular area.

I was also thinking we might want to include some template slide decks that less experienced speakers can use to get their talks rolling.

Some other ideas I had were being more assertive in-terms of agreeing and sharing speaker time limits in advance, then nominating someone to enforce these during the talks themselves. This would encourage speakers to be more considerate about the audiences times, raise the quality of the discussion after the talks, and hopefully encourage speakers to reflect on (and practice?) their talks in advance. We also got feedback from multiple members that clearly communicating and enforcing the length of our talks would be appreciated.

Another idea is to provide some support for speakers to practice and get feedback on their talks in advance of the meetup. This could mean providing a time to practice in person or online, or could just involve looking at slides or code and providing feedback.

It follows from providing practice time and resources to our speakers, that we need to plan as many of the talks as far in advance as possible. Ideally, I'd like to get at least a few talks for each meetup over the next year in place by September. Then we can move these around and fill out additional speakers as the meetups get closer, but speakers will at least have a lot of time to prepare.

To get this many speakers, we'd probably benefit from putting our members to work to volunteer to speak, but also to suggest people we should be approaching to see if they're interested in speaking. This could be through a quick form or something more permanent and on-going on the website.

I also see recording and sharing our talks as part of this. It sends a message to our presenters that we appreciate and care about the quality of our presentations, and increases the size of our audience to members not able to attend the meetup in person.

I'd also like to come up with some kind of metrics we can use to gauge the quality of our talks and size of our audience. If we can get recordings going, we can combine meetup attendance with video views to gauge audience size. Measuring quality will probably require doing some kind of survey at multiple points during the year? Any suggestions for doing this are welcome.

Do any other @ExchangeJS/organizers have ideas on how to increase the quality and audience of our talks?

**Comment by MarkBennett** [comment]
I just created issue #52 so that we remember to ask members who they'd like to see at the meetup.

**Comment by abramhindle** [comment]
Does this idea also include the 10 minutes of content for new javascript programmers? The Newbie 10?  

Providing an Markdown based Reveal JS slide deck template might help new speakers. Ben had a slide deck I found really useful.

**Comment by MarkBennett** [comment]
That would definitely be in scope for this @abramhindle. I'm going to make a list of possible actions for this up in the description so that we don't need to scroll back through all the comments.

**Comment by MarkBennett** [comment]
I've added the possible actions to the issue description. Please feel free to add any others you can think of then we'll whittle the list down.

**Comment by MarkBennett** [comment]
Based off of your suggested topics @CAYdenberg, I put together a draft talk schedule we can fill in for the year.

https://docs.google.com/spreadsheets/d/1_FOG1K_fUbTlNvqT9hjihiXYt9AhPkZxW6tfZN858VU/edit?usp=sharing

**Comment by MarkBennett** [comment]
The only other idea we've talked about that I haven't seen captured in this thread is having a specific beginner talk time, perhaps before the formal part of the meetup. I've heard the Ruby meetup had luck with setting aside an hour before the meetup started with more hands on time where experienced devs could assist younger devs. When I was running the Ruby meetup we also tried to have regular hackups where beginners could bring their laptops and ask questions.
